src/Configuration/ConfigurationFactory.php

```diff
@@ -64,6 +65,11 @@ private function canShowProgressBar(InputInterface $input): bool
             return false;
         }
 
+        $notCheckstyleOutput = $input->getOption(Option::OUTPUT_FORMAT) !== CheckstyleOutputFormatter::NAME;
+        if (! $notCheckstyleOutput) {
+            return false;
+        }
+
```

Comment:
There are 2 negatives, that are hard to read. Could you turn them positive compares?

Reply:
I will change them. 
Originally, I copied the [code from the JSON output condition below](https://github.com/easy-coding-standard/easy-coding-standard/blob/main/src/Configuration/ConfigurationFactory.php#L67-L70) and adjusted it for the checkstyle format.
```php
        $notJsonOutput = $input->getOption(Option::OUTPUT_FORMAT) !== JsonOutputFormatter::NAME;
        if (! $notJsonOutput) {
            return false;
        }
```

Reply:
I see. I wonder if we should use some better detection not to depend on specific strings to compare. 
Any ideas?

Reply:
To avoid over-complication and leave room for additional formatters, I would change it into a single `in_array` check.
```php
$outputFormat = $input->getOption(Option::OUTPUT_FORMAT);
$formatsWithoutProgressBar = [CheckstyleOutputFormatter::NAME, JsonOutputFormatter::NAME];
if (in_array($outputFormat, $formatsWithoutProgressBar, true)) {
    return false;
}
```

Reply:
Should I change it to this and push a new commit? Or rather amend my previous commit and force-push so it will only show up as one commit?

Reply:
> Should I change it to this and push a new commit?

That would be the best to review only new changes. Thanks